flink-runtime/src/test/scala/org/apache/flink/runtime/testingUtils/TestingUtils.scala

```diff
@@ -384,12 +434,12 @@ object TestingUtils {
    * @return
     */
   def createJobManager(
-      actorSystem: ActorSystem,
-      futureExecutor: Executor,
-      ioExecutor: Executor,
-      configuration: Configuration,
-      jobManagerClass: Class[_ <: JobManager],
-      prefix: String)
+                        actorSystem: ActorSystem,
+                        futureExecutor: ScheduledExecutorService,
+                        ioExecutor: Executor,
+                        configuration: Configuration,
+                        jobManagerClass: Class[_ <: JobManager],
+                        prefix: String)
```

Comment:
yeah, I think that way an IDE auto format action. Never figured out how to turn that off. Will undo that...